tests/WE2E/test_configs/wflow_features/config.specify_template_filenames.yaml

```diff
@@ -5,11 +5,11 @@ metadata:
 user:
   RUN_ENVIR: community
 workflow:
-  DATA_TABLE_TMPL_FN: data_table
-  DIAG_TABLE_TMPL_FN: diag_table.FV3_GFS_v15p2
-  FIELD_TABLE_TMPL_FN: field_table.FV3_GFS_v15p2
-  MODEL_CONFIG_TMPL_FN: model_configure
-  NEMS_CONFIG_TMPL_FN: nems.configure
+  DATA_TABLE_FN: data_table
+  DIAG_TABLE_FN: diag_table.FV3_GFS_v15p2
+  FIELD_TABLE_FN: field_table.FV3_GFS_v15p2
```

Comment:
@MichaelLueken This was related to the problem I mentioned yesterday: these variables are not named correctly in develop, and that was causing the new script to fail. However, this fix needs to be applied more carefully than this, so I have reverted these changes for now in order to avoid the failure you pointed out. I will open an issue later with more details to hopefully fix this.

Reply:
@mkavulich Thank you very much for the explanation and reverting this change for now.  I certainly agree that the entries for `DATA_TABLE_TMPL_FN`, `MODEL_CONFIG_TMPL_FN`, and `NEMS_CONFIG_TMPL_FN` appear to be set incorrectly with respect to the `specify_template_filenames` test.  As a matter of fact, there are no entries for either `MODEL_CONFIG_TMPL_FN` or `NEMS_CONFIG_TMPL_FN` in `config_defaults.yaml` and `data_table` would need to be changed to `data_table.FV3_GFS_v15p2` in order to properly be set to `DATA_TABLE_TMPL_FN`, so I can agree that these would likely cause the new script to fail.  I'll resolve this conversion for now, reapprove, and then rerun the Cheyenne tests.